articles/aks/api-server-vnet-integration.md

```diff
@@ -27,7 +27,7 @@ API Server VNet Integration is supported for public or private clusters. You can
 ## Limited availability
 
 > [!IMPORTANT]
-> **API Server VNet Integration has limited availability and capacity in certain regions.**  
+> **API Server VNet Integration is supported in all regions but may have limited capacity in certain regions.**  
 > When creating or updating a cluster with API Server VNet Integration enabled, you may receive the following error:
 >
 > **`API Server VNet Integration is currently unavailable in region (_region_) due to high demand and limited capacity. AKS is actively expanding support for this feature. Check for other available regions at aka.ms/AksVnetIntegration.`**
```

Comment:
Actually, I think the only thing we need to fix is the message here ` unavailable in region (_region_)`. This should be changed to something like `only available in limited regions per below list`.